openapi/access.yaml

```diff
@@ -933,11 +933,18 @@ components:
         - Executed
         - Sealed
         - Expired
+    BlockStatus:
+      type: string
+      description: This value indicates the status of a block.
+      enum:
+        - BLOCK_UNKNOWN
+        - BLOCK_FINALIZED
+        - BLOCK_SEALED
```

Comment:
for reference, these are defined [here](https://github.com/onflow/flow-go/blob/1ba41690954621d15d59dcabcc8f0dfc0b56f1bd/model/flow/block.go#L61-L75)